File: Python/pr_7.py
def main():
    # Read the size of the array
    size = int(input("Enter the size of the array: "))

    # Read the elements of the array
    arr = []
    print("Enter the elements of the array: ")
    for _ in range(size):
        arr.append(int(input()))

    # Create a new list to store the unique elements
    unique = []
    
    # Traverse the original list
    for element in arr:
        # Check if the element is already present in the new list
        if element not in unique:
            unique.append(element)

    # Print the unique elements
    print("Array with duplicate elements removed:", unique)

if __name__ == "__main__":
    main()

File: C/pr_7.c
#include <stdio.h>

int main() {
    int size, i, j, k;
    
    // Read the size of the array
    printf("Enter the size of the array: ");
    scanf("%d", &size);
    
    // Read the elements of the array
    int arr[size];
    printf("Enter the elements of the array: ");
    for (i = 0; i < size; i++) {
        scanf("%d", &arr[i]);
    }
    
    // Create a new array to store the unique elements
    int unique[size];
    int isDuplicate;
    int uniqueSize = 0;
    
    // Traverse the original array
    for (i = 0; i < size; i++) {
        // Check if the element is already present in the new array
        isDuplicate = 0;
        for (j = 0; j < uniqueSize; j++) {
            if (arr[i] == unique[j]) {
                isDuplicate = 1;
                break;
            }
        }
        
        // If it is not a duplicate, add it to the new array
        if (!isDuplicate) {
            unique[uniqueSize] = arr[i];
            uniqueSize++;
        }
    }
    
    // Print the unique elements
    printf("Array with duplicate elements removed: ");
    for (k = 0; k < uniqueSize; k++) {
        printf("%d ", unique[k]);
    }
    
    return 0;
}
